multiopython/plans/actions.py

```diff
@@ -67,7 +56,7 @@ class Print(Action):
 
     type: Literal["print"] = "print"
     stream: Literal["cout"] = Field("cout")
-    prefix: str = Field("")
+    prefix: str = Field(default_factory=lambda: "")
```

Comment:
strings are immutable in python (unless you go inside the c api), so for this one it's actually ok to go with just `prefix: str = ""`, no need for factory (which was returning the same literal anyway, not constructing it anew on every call)